Formula/s/simde.rb

```diff
@@ -13,6 +13,17 @@ class Simde < Formula
   depends_on "meson" => :build
   depends_on "ninja" => :build
 
+  # Patches iterated on via https://github.com/simd-everywhere/simde/issues/1146
```

Comment:
Asked upstream if they can land this as a single patch

Reply:
I think we can just ship as two patches, we have couple of examples of doing it before.